### ISSUE TITLE
Use isort with Jinja2 variables on python files

### ISSUE THREAD

**Issue by luzfcb** [timothycrosley/isort#250]
Hello, I am one of the co-maintainers of the [cookiecutter-django](https://github.com/pydanny/cookiecutter-django)[1], a [Cookiecutter](https://github.com/audreyr/cookiecutter)[2] based Django project template.

I would like to use the isort to test the order of imports of every new pull-requests

The Cookiecutter uses Jinja2 internally to define variables that will be replaced during the generation of new project

All the variables of a template project that uses cookiecuter are in the namespace `cookiecutter` like this:

```python
{{ cookiecutter.foo_variable }}
```

Eg, I have the following file:
```python
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import os
import sys


try:
    from setuptools import setup
except ImportError:
    from distutils.core import setup


from foo import {{ cookiecutter.bar }}
import {{ cookiecutter.repo_name }}
version = {{ cookiecutter.repo_name }}.__version__
```

after running `isort file.py` results in:
```python
#!/usr/bin/env python
# -*- coding: utf-8 -*-

import os
import sys

import cookiecutter.repo_name

import {{
import }}
from foo import cookiecutter.bar, {{, }}

try:
    from setuptools import setup
except ImportError:
    from distutils.core import setup


version = {{ cookiecutter.repo_name }}.__version__

```

I know I could just use "` # isort: skip` "to skip the lines that use Jinja2 the variable syntax, however, this is not an option, because I want isort working properly on the generated project.

The current implementation of isort allows some sort of customization, which might have escaped by me?

I think a possible implementation was to treat `{{ cookiecutter.foo_variable }}` as a single item.

What are your thoughts on this?

[1] https://github.com/pydanny/cookiecutter-django
[2] https://github.com/audreyr/cookiecutter

**Comment by timothycrosley** [comment]
Hi @luzfcb,

Thanks for letting me know! I would be more than happy to add support for these variable imports prior to the next release, and am happy to hear you would like to use isort for your project (a project I myself use).

Thanks!

~Timothy

**Comment by timothycrosley** [comment]
Hi @luzfcb,

This should be fixed in the 4.0 version of isort just released to PYPI. 

Thanks!

~Timothy
